src/Marten/Schema/JsonLocatorField.cs

```diff
@@ -63,28 +74,9 @@ public JsonLocatorField(string dataLocator, StoreOptions options, EnumStorage en
             }
         }
 
-        public JsonLocatorField(string dataLocator, EnumStorage enumStyle, Casing casing, MemberInfo[] members) : base(enumStyle, members)
+        public JsonLocatorField(string dataLocator, EnumStorage enumStyle, Casing casing, MemberInfo[] members) :
+            this(dataLocator, new StoreOptions(), enumStyle, casing, members, null)
```

Comment:
@barryhagan I think that for that case it'd be better to pass just `null`. Sending default StoreOptions might be misleading, as they don't relate with the StoreOptions that user have set up in `DocumentStore` initialization. Imho having null would be more clear indication that for this case we're not sending any options.

Reply:
done